include/psim/fc/orbit_controller.yml

```diff
@@ -0,0 +1,33 @@
+
+name: OrbitControllerInterface
+type: Model
+comment: >
+    Interface for how the flight computer's orbit controller will interact with the rest of the simulation
```

Comment:
If you don't have vertical rulers setup for your VSCode instance I'd recommend doing that [here](https://stackoverflow.com/questions/29968499/vertical-rulers-in-visual-studio-code)! We want to try and limit these files to an 80 character line limit especially for the comment sections so I'd go ahead and wrap the text at the top there.